intel_extension_for_transformers/llm/runtime/graph/application/main_pybind.cpp

```diff
@@ -55,13 +55,13 @@ class Model {
   ~Model() {
     if (ctx) model_free(ctx);
   }
-  void init_model(const std::string& model_path, int n_predict, int batch_size, int ctx_size, int seed, int threads,
+  void init_model(const std::string& model_path, int n_predict, int n_batch, int ctx_size, int seed, int threads,
```

Comment:
what's the differences between ```nbatch``` and ```batch_size```

Reply:
`n_batch` is the max tokens len for prompt process (`-b`). `batch_size` is just the inference bs